submissions/description/server_security_misconfiguration/misconfigured_dns/high_impact_subdomain_takeover/template.md

```diff
@@ -0,0 +1,23 @@
+# High-Impact Subdomain Takeover
+
+## Overview of the Vulnerability
+
+A subdomain takeover is when a misconfigured Domain Name System (DNS) record is re-registered to an endpoint owned by an attacker. An attacker is then able to redirect users to the endpoint and capture data such as cookies and credentials, perform Cross-Site Scripting (XSS) attacks, and potentially take over accounts in the legitimate application.
+
+A high-impact subdomain takeover vulnerability was identified which could impact the reputation and brand of the business. An attacker can register a subdomain on behalf of the target domain and use it for spamming and phishing attacks.
```

Comment:
Highlighting the technical impact will help clarify what the difference here is, at the end of the day we are trying to prove the researcher and customer a direct explanation of what a High Impact subdomain takeover.

Reply:
@evildaemond Please let me know if the additional context is sufficient